test_libs/pyspec/eth2spec/debug/decode.py

```diff
@@ -0,0 +1,28 @@
+from eth2spec.utils.minimal_ssz import hash_tree_root
+
+
+def decode(json, typ):
```

Comment:
Seems like overkill to have `encode` and `decode` in separate files

Reply:
I think it's good practice here, albeit the code being small. They are independent, and `encode` is used way more often than `decode` for test-generation.